pal_event_server/lib/src/daemon/linux/dbus_notifications.dart

```diff
@@ -44,19 +44,29 @@ const _defaultActions = <String>[
 ];
 
 // Map between Taqo database notification id and libnotify id
-final _notifications = <int, int>{};
+// visible for testing
```

Comment:
We can use the `@visibleForTesting` annotation (from meta package) here so that the Dart Analyzer will warn us if the method is accessed from code that is not in in the same library or in test/.

Reply:
Yes! Done.